src/qt/listctrl.cpp

```diff
@@ -2108,6 +2113,36 @@ bool wxListCtrl::SortItems(wxListCtrlCompare fn, wxIntPtr data)
     return true;
 }
 
+void wxListCtrl::ShowSortIndicator(int col, bool ascending)
+{
+    const auto header = m_qtTreeWidget->header();
+    if ( header )
+        header->setSortIndicator(col, ascending ? Qt::DescendingOrder
+                                                : Qt::AscendingOrder);
+}
+
```

Comment:
This should be
`        header->setSortIndicator(col, ascending ? Qt::AscendingOrder : Qt::DescendingOrder);`

But there seems to be a bug somewhere which is causing it to not work as expected!

Reply:
Do you mean that it works right now, with the inverted logic? If yes, then we must not be using the same meaning for the "ascending" order as Qt does and I guess we just need to add a comment documenting it here.

Reply:
Yes, with the inverted logic everything is ok.

After building a simple Qt program (using **QtCreator**), I discovered that the sort indicator symbol is indeed reversed!

I'll just add a comment documenting it then.